services/121-service/src/notifications/whatsapp/whatsapp-generic-message.json

```diff
@@ -0,0 +1,9 @@
+{
+  "ar": "{{1}}",
+  "es": "{{1}}",
+  "en": "This is a message from {{1}}.\n\nThere is a message waiting for you. Please reply “yes” to this message to receive it.",
```

Comment:
@RubenGeo you say here "Netherlands Red Cross" (from instance displayname) but in other messages we say "from the Red Cross". We should align that.